<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="pixmaps/imagens.qrc">
    <normaloff>:/img/appicon</normaloff>:/img/appicon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="pixmaps/imagens.qrc">:/img/esdas</pixmap>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="semaforoBD">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="pixmaps/imagens.qrc">:/img/desligado</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Base de Dados</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="semaforoTabela">
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="pixmaps/imagens.qrc">:/img/desligado</pixmap>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Tabela &quot;Produto&quot;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuBase_de_Dados">
    <property name="title">
     <string>Base de Dados</string>
    </property>
    <addaction name="actionLigarBD"/>
    <addaction name="actionDesligarBD"/>
   </widget>
   <widget class="QMenu" name="menuAjuda">
    <property name="title">
     <string>Ajuda</string>
    </property>
    <addaction name="actionAcerca"/>
   </widget>
   <addaction name="menuBase_de_Dados"/>
   <addaction name="menuAjuda"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLigarBD"/>
   <addaction name="actionCriar_tabela_de_Produtos"/>
   <addaction name="actionEditarTabelaProdutos"/>
   <addaction name="actionDesligarBD"/>
   <addaction name="actionSair"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLigarBD">
   <property name="icon">
    <iconset resource="pixmaps/imagens.qrc">
     <normaloff>:/img/on</normaloff>:/img/on</iconset>
   </property>
   <property name="text">
    <string>ligarBD</string>
   </property>
   <property name="toolTip">
    <string>Estabelecer ligação com a base de dados.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDesligarBD">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="pixmaps/imagens.qrc">
     <normaloff>:/img/off</normaloff>:/img/off</iconset>
   </property>
   <property name="text">
    <string>desligarBD</string>
   </property>
   <property name="toolTip">
    <string>Terminar a ligação com a base de dados.</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionSair">
   <property name="icon">
    <iconset resource="pixmaps/imagens.qrc">
     <normaloff>:/img/exit</normaloff>:/img/exit</iconset>
   </property>
   <property name="text">
    <string>sair</string>
   </property>
   <property name="toolTip">
    <string>Sair</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionAcerca">
   <property name="text">
    <string>Acerca ...</string>
   </property>
   <property name="toolTip">
    <string>Acreca do projecto ...</string>
   </property>
  </action>
  <action name="actionCriar_tabela_de_Produtos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="pixmaps/imagens.qrc">
     <normaloff>:/img/produto</normaloff>:/img/produto</iconset>
   </property>
   <property name="text">
    <string>Criar tabela de Produtos</string>
   </property>
  </action>
  <action name="actionEditarTabelaProdutos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="pixmaps/imagens.qrc">
     <normaloff>:/img/editar</normaloff>:/img/editar</iconset>
   </property>
   <property name="text">
    <string>EditarTabelaProdutos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="pixmaps/imagens.qrc"/>
 </resources>
 <connections/>
</ui>
